Makefile

```diff
@@ -957,6 +970,109 @@ topo/us-%-cities.json: geojson/%/cities.geojson
 		--properties name,scalerank,pop_max \
 		-- $<
 
+#
+# Albers-projected Vector Tiles
+#
+geojson/albers/state-centroids.geojson:
+	mkdir -p $(dir $@)
+	cat data/state-centroids.geojson \
+		| ./reproject-geojson \
+		| ./normalize-properties postal:statePostal name:name \
+		> $@
+
+geojson/albers/%.geojson: geojson/%.geojson
+	mkdir -p $(dir $@)
+	cat $^ \
+		| ./reproject-geojson \
+		| ./normalize-properties GEOID:id STATE_FIPS:id FIPS:id  \
+		> $@
+
+geojson/albers/state-bounds.json: geojson/albers/states.geojson
+	cat $^ | ./extract-projected-bounds > $@
+
+election-results/2012.csv:
+	mkdir -p $(dir $@)
+	data/ap-to-csv data/2012-county-results.json data/2012-district-results.json > $@
+
+election-results/2012-state-centroids.geojson: geojson/albers/state-centroids.geojson
+	cat geojson/albers/state-centroids.geojson \
+		| node_modules/.bin/geojson-join \
+			--format=csv --againstField=statePostal --geojsonField=statePostal election-results/2012.csv \
+		| ./flatten-geojson \
+		> $@
+
+election-results/2012-%.geojson: geojson/albers/%.geojson
+	cat geojson/albers/$*.geojson \
+		| node_modules/.bin/geojson-join \
+			--format=csv --againstField=id --geojsonField=id election-results/2012.csv \
+		| ./add-density-property voteCount \
+		| ./flatten-geojson \
+		> $@
+
+election-results/2012-%-10m.geojson: geojson/albers/us-10m/%.geojson
+	cat geojson/albers/us-10m/$*.geojson \
+		| node_modules/.bin/geojson-join \
+			--format=csv --againstField=id --geojsonField=id election-results/2012.csv \
+		| ./add-density-property voteCount \
+		| ./flatten-geojson \
+		> $@
```

Comment:
^ note the duplication here: 2012-%-10m.geojson is analogous to 2012-%.geojson, but derived from the simplified version of the boundary file.